bedrock/firefox/templates/firefox/android/index-b.html

```diff
@@ -0,0 +1,43 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. #}
+
+{% extends "firefox/android/index.html" %}
+
+{% block email_form %}
+  <div id="subscribe-download-wrapper" class="ga-section" data-ga-label="Get Firefox news">
```

Comment:
Do we need to repeat all of this template markup? As far as I can tell, only the newsletter title changes on this page. If so (I could be wrong), could we just create a new block in `index.html` that wraps only the newsletter marco, and overwrite that here instead?